src/components/Search/Search.js

```diff
@@ -21,6 +22,40 @@ function Search() {
   const searchService = useSelector(state => state.app.searchService);
   const { t } = useTranslation();
   const dispatch = useDispatch();
+  const renderInputComponent = inputProps => {
+    return (
+      <div className="wkp-search-input">
+        <input {...inputProps} />
```

Comment:
Eslint complains about not spreading props, but this is the way React-AutoSuggest defines. Maybe there is a better way though

Reply:
no it's ok for this one. you can deactivate it